File: plop-templates/adapter/payout/src/handler/CallbackHandler.java
package com.rbkmoney.{{adapterPayoutPackageCase bank_name}}.handler;

import com.rbkmoney.adapter.bank.payout.spring.boot.starter.handler.HandleCallbackHandler;
import com.rbkmoney.damsel.msgpack.Value;
import com.rbkmoney.damsel.withdrawals.provider_adapter.Callback;
import com.rbkmoney.damsel.withdrawals.provider_adapter.CallbackResult;
import com.rbkmoney.damsel.withdrawals.provider_adapter.Withdrawal;
import org.springframework.stereotype.Component;

import java.util.Map;

@Component
public class CallbackHandler implements HandleCallbackHandler {

    @Override
    public CallbackResult handleCallback(Callback callback, Withdrawal withdrawal, Value value,
                                         Map<String, String> map) {
        throw new RuntimeException("Not Supported handle");
    }
}

File: plop-templates/adapter/payout/src/handler/CallbackHandler.kt
package com.rbkmoney.{{adapterPayoutPackageCase bank_name}}.handler

import com.rbkmoney.adapter.bank.payout.spring.boot.starter.handler.HandleCallbackHandler
import com.rbkmoney.damsel.msgpack.Value
import com.rbkmoney.damsel.withdrawals.provider_adapter.Callback
import com.rbkmoney.damsel.withdrawals.provider_adapter.CallbackResult
import com.rbkmoney.damsel.withdrawals.provider_adapter.Withdrawal
import org.springframework.stereotype.Component

@Component
class CallbackHandler : HandleCallbackHandler {
    override fun handleCallback(
        callback: Callback,
        withdrawal: Withdrawal,
        value: Value,
        map: Map<String, String>
    ): CallbackResult {
        throw RuntimeException("Not Supported handle")
    }
}
